File: docs/datatools/codesnippet/CSharp/query-datasets_4.cs
            string custID = "ALFKI";
            NorthwindDataSet.OrdersRow[] orders;

            orders = (NorthwindDataSet.OrdersRow[])northwindDataSet.Customers.
                FindByCustomerID(custID).GetChildRows("FK_Orders_Customers");

            MessageBox.Show(orders.Length.ToString());

File: docs-old-1103-2/data-tools/codesnippet/VisualBasic/query-datasets_4.vb
        Dim customerID As String = "ALFKI"
        Dim orders() As NorthwindDataSet.OrdersRow

        orders = CType(NorthwindDataSet.Customers.FindByCustomerID(customerID).
            GetChildRows("FK_Orders_Customers"), NorthwindDataSet.OrdersRow())

        MessageBox.Show(orders.Length.ToString())